### PR TITLE
Validation Rules Result Screen - valid (EXPOSUREAPP-8031)

### PR DESCRIPTION
**Pull request by BMItr**
This PR addresses EXPOSUREAPP-8031 (validation Result Screen - valid)
PR will be set to **DRAFT** because Button & Date - Wiring needed after Repo and Screens before this one are completed/merged (see EXPOSUREAPP-7738 for the flow)

Screenshot (scrolled down a little):

screen

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml [hunk 1]
```diff
@@ -102,4 +102,34 @@
...
     <!-- XTXT: Test certificate card name  -->
     <string name="test_certificate_name">Testzertifikat</string>
+
+    <!-- #####################################################
+      VALIDATION RULES RESULT SCREEN (VALID) STRINGS BELOW
+      ##################################################### -->
+    <!-- XTXT: Validation Rule Result Valid - Header  -->
+    <string name="validation_rules_result_valid_header">Gültiges Zertifikat</string>
+    <!-- XTXT: Validation Rule Result Valid - Result  -->
+    <string name="validation_rules_result_valid_result_title">Ihr Zertifikat ist im gewählten Reiseland gültig.</string>
```
Comment (ChristinaTsigganos) [inline Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml:112]: Please change "Reiseland" to "Land", as displayed in Figma: https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.2?node-id=24596%3A21940

File: Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml [hunk 2]
```diff
@@ -101,5 +101,35 @@
...
\ No newline at end of file
+    <string name="test_certificate_name">Test Certificate</string>
+
+    <!-- #####################################################
+      VALIDATION RULES RESULT SCREEN (VALID) STRINGS BELOW
+      ##################################################### -->
+    <!-- XTXT: Validation Rule Result Valid - Header  -->
+    <string name="validation_rules_result_valid_header">Gültiges Zertifikat</string>
+    <!-- XTXT: Validation Rule Result Valid - Result  -->
+    <string name="validation_rules_result_valid_result_title">Ihr Zertifikat ist im gewählten Reiseland gültig.</string>
```
Comment (ChristinaTsigganos) [inline Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml:112]: same here, please change "Reiseland" to "Land".

**Review by ChristinaTsigganos — CHANGES_REQUESTED** [review]
Please adjust "Reiseland" to "Land", apart from that the texts are approved by UA

**Comment by vaubaehn** [comment]
@harambasicluka @ChristinaTsigganos 

I'd suggest a small text enhancement for the 4th bullet point (even the text might also still be draft):

`Ob die im Zertifikat eingetragenen Daten richtig sind, wird `**von der Corona-Warn-App**` nicht geprüft.`

It's because the first three bullet points deal "with things outside the scope" of CWA. Especially the 3rd point is handling with "a verifier app". Hence, at least for me, the last bullet sounds like the verifier app won't check the contents of the certificate. Adding "von der Corona-Warn-App" clears it up imho, and hopefully _all_ users get it right then...

**Comment by harambasicluka** [comment]
> @harambasicluka @ChristinaTsigganos
> 
> I'd suggest a small text enhancement for the 4th bullet point (even the text might also still be draft):
> 
> `Ob die im Zertifikat eingetragenen Daten richtig sind, wird `**von der Corona-Warn-App**` nicht geprüft.`
> 
> It's because the first three bullet points deal "with things outside the scope" of CWA. Especially the 3rd point is handling with "a verifier app". Hence, at least for me, the last bullet sounds like the verifier app won't check the contents of the certificate. Adding "von der Corona-Warn-App" clears it up imho, and hopefully _all_ users get it right then...
> 
> Edit: Or move the 4th bullet to the beginning as first bullet, and write as
> `Ob die im Zertifikat eingetragenen Daten richtig sind, wird `**hier**` nicht geprüft.` Then the coherence of the following three bullets fits also well.

Thanks @vaubaehn we are looking into this 👍

**Comment by ChristinaTsigganos** [comment]
> @harambasicluka @ChristinaTsigganos
> 
> I'd suggest a small text enhancement for the 4th bullet point (even the text might also still be draft):
> 
> `Ob die im Zertifikat eingetragenen Daten richtig sind, wird `**von der Corona-Warn-App**` nicht geprüft.`
> 
> It's because the first three bullet points deal "with things outside the scope" of CWA. Especially the 3rd point is handling with "a verifier app". Hence, at least for me, the last bullet sounds like the verifier app won't check the contents of the certificate. Adding "von der Corona-Warn-App" clears it up imho, and hopefully _all_ users get it right then...
> 
> Edit: Or move the 4th bullet to the beginning as first bullet, and write as
> `Ob die im Zertifikat eingetragenen Daten richtig sind, wird `**hier**` nicht geprüft.` Then the coherence of the following three bullets fits also well.

Thanks a lot, @vaubaehn. We checked and decided to consider your first suggestion to add the information  "von der Corona-Warn-App" to the last bullet point. Thanks a lot for your valuable contribution to the CWA.

**Comment by BMItr** [comment]
This PR will probably be closed and replaces by another due new Common-Model (new Base Branch).
Will clean it when its done.
No more review required here.

**Comment by BMItr** [comment]
Follow UP: https://github.com/corona-warn-app/cwa-app-android/pull/3678

**Pull request closed without merging**